package.json

```
@@ -33,9 +33,9 @@
     "clean": "npm-run-all -p clean:*",
     "clean-and-develop": "pnpm run clean && pnpm install && pnpm run develop",
     "clean:client": "cd ./client && pnpm run clean",
-    "clean:curriculum": "npx --yes rimraf ./config/curriculum.json",
-    "clean:packages": "npx --yes rimraf ./**/node_modules",
-    "clean:server": "npx --yes rimraf ./api-server/lib",
+    "clean:curriculum": "rimraf ./config/curriculum.json",
+    "clean:packages": "rimraf ./**/node_modules node_modules",
```

```suggestion
    "clean:packages": "rimraf ./node_modules ./**/node_modules",
```

This should make it not look like a typo later in future